spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinderFactory.java

```diff
@@ -50,9 +51,12 @@
  * @author Ilayaperumal Gopinathan
  * @author Gary Russell
  * @author Oleg Zhurakousky
+ * @author Soby Chacko
  */
 public class DefaultBinderFactory implements BinderFactory, DisposableBean, ApplicationContextAware {
 
+	private static ConfigurableApplicationContext OUTER_CONTEXT;
```

Comment:
Doesn't look like this property is in use.
And that's is good because `static` shared property is evil for Spring 😄